### DISCUSSION TITLE
Support for Q7872, oneway:conditional

### DISCUSSION THREAD

**Discussion by palzkillm** [organicmaps/organicmaps#4698] [Ideas]
Hello,
today I found, that Sierichstraße & Herbert-Weichmann-Straße, Hamburg, Germany, is changing direction daily between 4 AM and 12 AM. This is stored in OSM, using https://wiki.openstreetmap.org/wiki/Item:Q7872, with "yes @ (12:00-04:00);-1 @ (04:00-12:00)".

OM doesnt seem to be recognizing this, thus sending me into the wrong direction. I'd suggest to add this feature.

Marco

**Comment by biodranik** [comment]
Can you please create an issue for it?
